packages/server/lib/browsers/index.ts

```diff
@@ -62,32 +71,31 @@ const setFocus = async function () {
   }
 }
 
-const getBrowserLauncher = function (browser) {
+function getBrowserLauncher (browser: Browser, browsers: FoundBrowser[]): BrowserLauncher {
   debug('getBrowserLauncher %o', { browser })
-  if (!isBrowserFamily(browser.family)) {
-    debug('unknown browser family', browser.family)
-  }
 
   if (browser.name === 'electron') {
-    return require('./electron')
+    return require('./electron') as typeof import('./electron')
```

Comment:
Not sure if this breaks things but should be able to do something like (I think)

```js
return await import('./electron')
```

And get typing without `as typeof`. Also good to standardize on one syntax. Then `getBrowserLauncher` will return a `Promise` and well need to be awaited, though.

Or, we can just do

```ts
import chrome from './chrome'
```

Etc all at the top - not sure if there's any performance implication here, but moving towards proper ESM seems like a good idea, so maybe one day we can ship native ESM instead of CommonJS (although this is likely many years away, it would still be nice to work towards it).

Reply:
Since each browser requires different dependencies, it's done this way for performance. Whether or not it makes a big (or indeed, positive) impact... I'm not sure, but I'm not going to refactor the lazy-loading in this PR. I did switch to await import: ce151923dab8d8e535bcc4a7be87f49de71db95